Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelToDetVol.hpp

```diff
@@ -0,0 +1,32 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#pragma once
+
+#include "Acts/Detector/DetectorVolume.hpp"
+#include "Acts/Detector/GeometryIdGenerator.hpp"
+#include "Acts/Geometry/CuboidVolumeBounds.hpp"
+#include "Acts/Geometry/CutoutCylinderVolumeBounds.hpp"
+#include "Acts/Geometry/CylinderVolumeBounds.hpp"
+#include "Acts/Geometry/GeometryContext.hpp"
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Geometry/TrapezoidVolumeBounds.hpp"
+
+#include "GeoModelKernel/GeoShape.h"
+
+namespace Acts {
+namespace GeoModelToDetVol {
+/// @brief Convert a GeoModel shape to a DetectorVolume
+///
+/// @param shape the GeoModel shape
+/// @param transform the transform to be applied
+/// @return the DetectorVolume
+std::shared_ptr<Experimental::DetectorVolume> convert(
+    const GeometryContext& context, const GeoShape& shape,
+    const std::string& name, const Transform3& transform);
```

Comment:
`GeometryContext`, `GeoShape` and `Transform3` can be forward declared, which then also allows you to remove some includes above.